poolqc/pooling_qc_sample.py

```diff
@@ -0,0 +1,408 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import dash.dependencies as dep
+import dash_table as dt
+import pandas
+import plotly.plotly as plotly
+import plotly.graph_objs as go
+import numpy as np
+
+
+pandas.options.mode.chained_assignment = None
+index = pandas.read_hdf('../data/rnaseqqc_cache.hd5')
```

Comment:
This should be `./data/rnaseqqc_cache.hd5` (note the single `.`), as per the other modules in this project. I've created ticket #25 which addresses this issue, but there's no need to add a fix to #25 to this PR.

Reply:
Please update this.

Reply:
Please update this.